give/src/components/Image/Image.js

```diff
@@ -0,0 +1,64 @@
+// Based on v3
+// https://www.aemcomponents.dev/content/core-components-examples/library/core-content/image.html
+
+import "./Image.css";
+import lava1 from "../../assets/lava1.jpeg";
+
+const LinkWrapper = ({ href, children }) => {
+  return href ? (
+    <a className="cmp-image__link" href={href}>
+      {children}
+    </a>
+  ) : (
+    children
+  );
+};
+
+export const Image = ({
+  src = "", // string
+  alt = "", // string
+  caption = "", // string
+  href = "", // string - URL
+  popupTitle = true, // boolean
+  classes = "", // string
+}) => {
+  if (src === "" && alt === "") return null;
+
+  return (
+    <div className={`image cru-image${classes ? " " + classes : ""}`}>
+      <div className="cmp-image">
+        <LinkWrapper href={href}>
```

Comment:
I know this is a first pass, but what users think a link should be an what is reality usually differs. I do a wrapper around links in order to add https:// onto just in case the user just types www.facebook.com kind of thing. Here's an example of the code
```
    if (!url.includes("http") && !url.includes("https")) {
      url = "https://" + url;
    }

Reply:
That's a good precaution. It lines up with how I typically aim to protect users from themselves.

Reply:
@LizzyRandall I'm revisiting your comments and looking at this one, I'm actually not going to implement it. These link URLs should be generated by AEM and in case it gives me a relative URL, I don't want to accidentally screw it up by adding my own protocol.

Reply:
@j2trumpet Mine is manually entered by the user so it can definitely be random. AEM generating I'm guessing would have the right URL 👍